daemon/algod/api/algod.oas2.json

```
@@ -3479,6 +3517,10 @@
           "description": "Lifts limits on log opcode usage during simulation.",
           "type": "boolean"
         },
+        "allow-unnamed-resources": {
+          "description": "Allow access to unnamed resources during simulation.",
```

Very nitpicky, but it seems the surrounding options use:
```suggestion
          "description": "Allows access to unnamed resources during simulation.",
```

---

Accepted in f343aa459